### DISCUSSION TITLE
Does setting 'enableSearch' to false on chosen columns have a performance impact?

### DISCUSSION THREAD

**Discussion by femto-code** [ujjwalguptaofficial/JsStore#304] [General]
As the documentation states, the `enableSearch` property is set to `true` per default. If I understand it correctly, this has the following implications:
1. For each column that has this property there is another index created - correct?
2. Speeding up database searches but at the same time implying extra disk space usage.
3. Making INSERT / UPDATE / DELETE queries slower than without index - **is this performance impact noticeable**? Let's say with 1000 records?

Thanks for your thoughts on this!

**Comment by ujjwalguptaofficial** [comment]
for 1000 or event with 10k performance wont be noticable. but yes indexing takes extra disk usage which you can remove if not required. But make sure that you won't need to search(use where uery) in future for that column.

**Reply by femto-code** [reply]
Thanks for the clarification!
